config/routes.rb

```diff
@@ -278,6 +262,9 @@ def not_implemented
     end
   end
 
-  match '*path', via: :all, to: not_implemented
-
+  if Feature.active?(:redirects)
+    match '*path', via: :all, to: 'catchall#not_found'
```

Comment:
I think this will break the ability to find old tools on the public website. 
Secondly, I'm not sure about the action name ```not_found```.